packages/collector/test/tracing/cloud/gcp/storage/test.js

```diff
@@ -50,7 +53,11 @@ if (
 } else {
   let mochaSuiteFn;
 
-  if (!supportedVersion(process.versions.node) || !process.env.GCP_PROJECT) {
+  if (
+    !supportedVersion(process.versions.node) ||
+    !process.env.GCP_PROJECT ||
+    semver.lte(process.versions.node, '12.0.0')
```

Comment:
_nitpick:_ I think using `lt` instead of `lte` would make the intend to ignore all 10.x versions a bit clearer. Not important though, it only makes a difference when running the tests exactly with 12.0.0 (in which case we would skip it).

Reply:
Yes must be `lt` 👍